purchases/src/main/kotlin/com/revenuecat/purchases/CustomerInfo.kt

```diff
@@ -50,9 +52,38 @@ data class CustomerInfo(
     val originalAppUserId: String,
     val managementURL: Uri?,
     val originalPurchaseDate: Date?,
+
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    val virtualCurrencies: Map<String, VirtualCurrencyInfo>,
     private val jsonObject: JSONObject,
 ) : Parcelable, RawDataContainer<JSONObject> {
 
+    @Deprecated("Use constructor with virtualCurrencies parameter")
+    constructor(
```

Comment:
This deprecates the old constructor to keep source compatibility

Reply:
Since the new constructor requires `@ExperimentalPreviewRevenueCatPurchasesAPI`, maybe it'd be a good idea to not deprecate this one yet, keep it as an alternate constructor, and then deprecate it if/when the constructor with the virtual currencies has the experimental annotation removed